cli/jobs/pipelines/automl/forecasting-energy-demand-pipeline/components/src_register/register.py

```diff
@@ -0,0 +1,92 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import argparse
+import json
+import os
+import time
+
+
+from azureml.core import Run
+
+import mlflow
+import mlflow.sklearn
+
+# Based on example:
+# https://docs.microsoft.com/en-us/azure/machine-learning/how-to-train-cli
+# which references
+# https://github.com/Azure/azureml-examples/tree/main/cli/jobs/train/lightgbm/iris
+
+
+def parse_args():
+    # setup arg parser
+    parser = argparse.ArgumentParser()
+
+    # add arguments
+    parser.add_argument("--model_input_path", type=str, help="Path to input model")
+    parser.add_argument(
+        "--model_info_output_path", type=str, help="Path to write model info JSON"
+    )
+    parser.add_argument(
+        "--model_base_name", type=str, help="Name of the registered model"
+    )
+    parser.add_argument(
+        "--model_name_suffix", type=int, help="Set negative to use epoch_secs"
+    )
+
+    # parse args
+    args = parser.parse_args()
+
+    # return args
+    return args
+
+
+def main(args):
+    current_experiment = Run.get_context().experiment
```

Comment:
Is this the recommended way to get the tracking uri etc in v2?

Reply:
@harneetvirk @paulshealy1 Cold you please take a look at this code?

It does not seem straightforward to access the model and register it.